cmd/repl/output.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2020 The SQLFlow Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/c-bata/go-prompt"
+	"strings"
+)
+
+// stdoutWriter is a prompt.ConsoleWriter implementation for SQLFlow.
+type stdoutWriter struct {
+	prompt.ConsoleWriter
+	currentBgColor prompt.Color
+	currentFgColor prompt.Color
+}
+
+// TODO(shendiaomo): get a full list of SQL keywords according to the dialect
+var keywords = map[string]bool{
+	"ALTER":     true,
+	"COLUMN":    true,
+	"CREATE":    true,
+	"DATABASE":  true,
+	"DATABASES": true,
+	"DESC":      true,
+	"DESCRIBE":  true,
+	"EXPLAIN":   true,
+	"FROM":      true,
+	"INSERT":    true,
+	"INTO":      true,
+	"LIMIT":     true,
+	"SELECT":    true,
+	"SHOW":      true,
+	"TABLE":     true,
+	"TABLES":    true,
+	"TO":        true,
+	"UPDATE":    true,
+	"USE":       true,
+	"USING":     true,
+	"VALUES":    true,
+	"WHERE":     true,
+	"WITH":      true,
+}
+var extendedKeywords = map[string]bool{
+	"TRAIN":   true,
+	"PREDICT": true,
+	"LABEL":   true,
+}
+
+// newStdoutWriter returns a ConsoleWriter object to write to stdout.
+func newStdoutWriter() *stdoutWriter {
+	return &stdoutWriter{ConsoleWriter: prompt.NewStdoutWriter()}
+}
+
+// WriteStr writes safety string by removing control sequences and highlighting keywords.
+func (w *stdoutWriter) WriteStr(data string) {
+	words := strings.Split(data, " ")
```

Comment:
maybe we should support '\t' as well

Reply:
`\t` is now a special character that activates auto-complete, users cannot input `\t` directly.